crates/network-scanner/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "network-scanner"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+network-scanner-net ={ path = "../network-scanner-net" }
+network-scanner-proto ={ path = "../network-scanner-proto" }
+tokio = { version = "1.35.1", features = ["io-util"] }
+tracing = "0.1.40"
+
+[dev-dependencies]
+tokio = { version = "1.35.1", features = ["full"] }
```

Comment:
suggestion: Enable only the required features instead of `full`.

Reply:
updated